src/dataset-builder/CohortEditor.test.ts

```diff
@@ -31,6 +31,13 @@ jest.mock('src/libs/ajax/DataRepo', (): DataRepoExports => {
   };
 });
 
+jest.mock('src/components/input', () => {
```

Comment:
Is this mock still necessary?

Reply:
I thought that since the tests were passing they were needed but I removed the mock and the tests are still passing.